index.js

```diff
@@ -147,46 +187,55 @@ const line = []
 
 // ***** Scope & Closures - Question 1 *****
 
+function takeATicketNumber(line){
+  counter = 0
+  for (counter = 0; counter < line.length; counter++){
+    console.log(`Welcome. You are thicket number ${line[0] + 1}`)
+    counter += 1
+  }
+}
```

Comment:
This isn't quite right, we'd want something like this - so when we run the function, a new number gets added to the line.

```js
let counter = 0

function takeATicketNumber(){
  counter += 1
  console.log(`Welcome. You are ticket number ${counter}`)
  line.push(counter)
}
```